geowebcache/core/src/main/java/org/geowebcache/config/XMLConfiguration.java

```diff
@@ -489,6 +488,8 @@ public static XStream getConfiguredXStreamWithContext(XStream xs, WebApplication
 
         xs.alias("serviceInformation", ServiceInformation.class);
         xs.alias("contactInformation", ContactInformation.class);
+
+        xs.omitField(ServiceInformation.class, "citeCompliant");
```

Comment:
Why omit?

Reply:
This one is a bit weird indeed. The ``ServiceInfo`` object is used in two distinct situations:
- storing GWC service information (common to WMTS and WMS)
- allow extension points to contribute more information to a specific service, for GeoServer provides specific information for WMTS service

I made CITE compliance be manageable per service, in this case for WMTS service only:
```
<?xml version="1.0" encoding="utf-8"?>
<gwcConfiguration xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
  xmlns="http://geowebcache.org/schema/1.13.0"
  xsi:schemaLocation="http://geowebcache.org/schema/1.13.0 http://geowebcache.org/schema/1.13.0/geowebcache.xsd">
  <version>1.13.0</version>
  <backendTimeout>120</backendTimeout>
  <wmtsCiteCompliant>true</wmtsCiteCompliant>
```

But the only way extension points have to contribute that information (CITE compliant mode activate \ deactivate) is through the service info object.

The ideal solution where will be to make possible for each service (WMTS, WMS, etc ...) have their own specific configuration, but this goes beyond the scope of this work.

Reply:
I see... yeah, rolling a new set of config files, one per service, seems indeed a lot of work.